spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm do
+  let(:ems) do
+    uid_ems  = "473f85b4-c4ba-4425-b495-d26c77365c91"
+    auth_key = Rails.application.secrets.ibmcvs.try(:[], :api_key) || "IBMCVS_API_KEY"
```

Comment:
You're not using any VCRs in this spec so you don't need to set the uid_ems/auth_key or any authentications